wic/orangepi-r1plus-imagedef.wks.in

```diff
@@ -5,10 +5,8 @@
 
 # 0~32K: gpt
 bootloader --ptable gpt
-part --source rawcopy --sourceparams="file=idblock.img" --align 32 --no-table
-part --source rawcopy --sourceparams="file=uboot.img" --part-name uboot --align 8192
-part --source rawcopy --sourceparams="file=trust.img" --part-name trust
-part /boot --ondisk mmcblk0 --source bootimg-partition --fstype ext2 --part-name boot
-part --source rootfs --fstype ${RK_ROOTFS_TYPE} --part-name rootfs-banka --uuid ${ORANGEPI_BANK_A_PART_UUID} --align 8192 --fixed-size 7168M
-part --fstype ${RK_ROOTFS_TYPE} --part-name rootfs-bankb --uuid ${ORANGEPI_BANK_B_PART_UUID} --align 8192 --fixed-size 7168M
-part /data --ondisk mmcblk0 --fstype ${RK_ROOTFS_TYPE} --part-name data --uuid ${ORANGEPI_DATA_PART_UUID} --align 8192 --fixed-size 1024M
+part --source rawcopy --sourceparams="file=idblock.img" --align 32 --no-table --fixed-size 4M
+part --source rawcopy --sourceparams="file=uboot.img" --part-name uboot --align 8192 --fixed-size 32M
+part --source rawcopy --sourceparams="file=trust.img" --part-name trust --fixed-size 32M
+part /boot --ondisk mmcblk0 --source bootimg-partition --fstype ext2 --part-name boot --fixed-size 256M
+part --source rootfs --fstype ${RK_ROOTFS_TYPE} --part-name rootfs-banka --uuid ${ORANGEPI_BANK_A_PART_UUID} --align 8192 --fixed-size 4092M
```

Comment:
Can we remove the A in this. I think it might cause misunderstanding here.
If A is here where is B, kinda things